var/spack/repos/builtin/packages/grass/package.py

```diff
@@ -195,3 +195,13 @@ def configure_args(self):
             args.append('--without-geos')
 
         return args
+
+    # see issue: https://github.com/spack/spack/issues/11325
+    # 'Platform.make' is created after configure step
+    # hence invoke the following function afterwards
+    def fix_iconv_linking(self):
+        makefile = FileFilter('include/Make/Platform.make')
+        makefile.filter(r'^ICONVLIB\s*=\s*', 'ICONVLIB = -liconv')
+        return None
+
+    run_after('configure')(fix_iconv_linking)
```

Comment:
This form isn't very common in Spack packages, you should add:
```
@run_after('configure')
```
to the line directly above `def fix_iconv_linking` instead.

Reply:
Good to know! Thanks.